### DISCUSSION TITLE
controls that don't rely on hass

### DISCUSSION THREAD

**Discussion by nguyenquangduc** [DomiStyle/esphome-panasonic-ac#93] [General]
can i use the configurations in the esphome climate component to control the axis directly without going through hass. eg:
- climate.control:
    id: my_climate
    mode: HEAT_COOL
    target_temperature: 25°C

**Comment by DomiStyle** [comment]
Can you clarify what you mean exactly? Do you mean controlling an AC without Home Assistant?

You can use any method ESPHome supports to control the AC: https://esphome.io/#core-components

This includes MQTT, the ESPHome API, a web interface or CAN bus.
